_dashboards/im-dashboards/component-templates.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Component templates
+parent: Index management in Dashboards
+nav_order: 50
+---
+
+# Component templates
+Introduced 2.7
+{: .label .label-purple }
+
+Component templates allow you to create a single index pattern that matches multiple indexes. This pattern can include wildcards or regular expressions, enabling you to apply the same setting or mapping to multiple indexes simultaneously.
+
+Using them with [index templates]({{site.url}}{{site.baseurl}}/im-plugin/index-templates/) can be a powerful tool for managing large volumes of data. You can create an index template that defines the basic structure and settings of your indexes, and then use the component templates to apply the settings to all indexes that match a specific pattern or criteria.
+
+You can create component templates from the Index Management user interface (UI). The UI maximizes ease of use for common indexing and data stream administrative operations such as create, read, update, delete (CRUD) and mapping indexes; CRUD and mapping aliases; reindexing; and open/close, shrink, and split indexes, along with the monitoring of actions and logging of audit records. 
```

Comment:
UI doesn't need to be defined: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#acronyms